certora/funnels/automation/StableDepositorUniV3.spec

```diff
@@ -0,0 +1,509 @@
+// StableDepositorUniV3.spec
+
+methods {
+    function wards(address) external returns (uint256) envfree;
+    function buds(address) external returns (uint256) envfree;
+    function configs(address, address, uint24, int24, int24) external returns (int32, uint32, uint96, uint96, uint96, uint96, uint32) envfree;
+    function _.deposit(DepositorUniV3Like.LiquidityParams) external => depositSummary() expect uint128, uint256, uint256;
+    function _.withdraw(DepositorUniV3Like.LiquidityParams, bool) external => withdrawSummary() expect uint128, uint256, uint256, uint256, uint256;
+    function _.collect(DepositorUniV3Like.CollectParams) external => collectSummary() expect uint256, uint256;
+}
+
+ghost uint128 retValue;
+ghost uint256 retValue2;
+ghost uint256 retValue3;
+ghost uint256 retValue4;
+ghost uint256 retValue5;
+
+function depositSummary() returns (uint128, uint256, uint256) {
+    return (retValue, retValue2, retValue3);
+}
+
+function withdrawSummary() returns (uint128, uint256, uint256, uint256, uint256) {
+    return (retValue, retValue2, retValue3, retValue4, retValue5);
+}
+
+function collectSummary() returns (uint256, uint256) {
+    return (retValue2, retValue3);
+}
+
+// Verify correct storage changes for non reverting rely
+rule rely(address usr) {
+    env e;
+
+    address other;
+    require other != usr;
+    address anyAddr;
+    address anyAddr2;
+    uint24 anyUint24;
+    int24 anyInt24;
+    int24 anyInt242;
```

Comment:
nit(naming): I might call this `anyInt24_2` just so it doesn't look like a numerical type with a weird width (applies throughout)

Reply:
Resolving this one.